OutOfSchool/OutOfSchool.WebApi/Models/Workshops/WorkshopBaseDto.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
-using Castle.Core.Internal;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
```

Comment:
Is this using required? Because no changes were introduced in the code, just new using added, and it seems to be irrelevant (as are irrelevant `identity tokens` and `workshop dto`)

Reply:
In Microsoft.IdentityModel.Tokens we have extension IsNullOrEmpty() for IEnumerable<T>. We can write this method in our extensions.

Reply:
Thank you. So recently it was in `Castle.Core.Internal`, now in `Microsoft.IdentityModel.Tokens`. 
I still suggest either write own implementation or find a related good library with collections extension methods.

Reply:
Our implementation was added.